django_filters/fields.py

```diff
@@ -92,6 +92,7 @@ def __init__(self, field, lookup_choices, *args, **kwargs):
         }
         widget = LookupTypeWidget(**defaults)
         kwargs['widget'] = widget
+        kwargs['help_text'] = field.help_text
```

Comment:
With this in place, then help_text defined for a filter will be displayed in the auto-generated documentation when lookup_expr is also provided.  This is slightly different from the issue you raised about using help_text from the model since this help_text is defined as part of the filter and can be worded appropriately.